src/Databases/MySql.php

```diff
@@ -100,6 +103,16 @@ public function setDefaultCharacterSet(string $characterSet)
         return $this;
     }
 
+    /**
+     * @return $this
+     */
+    public function setGtidPurged()
```

Comment:
Can this be named better?

Seems confusing from the outside that this method name is named positively, while the option is something negative `--set-gtid-purged=OFF`. 

Something like `gtidWasPurged` sounds better to me.

Reply:
Feel free to propose something even better.

Reply:
Looking back on iit, it has 3 states, ON, OFF and AUTO. [Source](https://dev.mysql.com/doc/refman/5.7/en/mysqldump.html#option_mysqldump_set-gtid-purged).
Will try to name it properly and add the rest of the functionality.